shell/common/BUILD.gn

```diff
@@ -172,6 +172,13 @@ source_set("common") {
         "//third_party/updater/target/x86_64-apple-darwin/release/libupdater.a",
       ]
     }
+  } else if (host_os == "win" && target_os == "win") {
+    if (target_cpu == "x64") {
+      libs = [
+        "userenv.lib",
```

Comment:
Is userenv still needed?

Reply:
afaik, yes. I don't think we've made any changes that would make this unnecessary